tsdb/head.go

```diff
@@ -231,6 +235,26 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 	return m
 }
 
+const cardinalityCacheExperationTime int64 = 30 //seconds
```

Comment:
s/Experation/Expiration

Since this is only used in one place it could be inlined if you wanted.

Reply:
Also if the unit is not clear it should be embedded in variable name OR `time.Duration` should be used. Comment can be lost easily. I would vote for `time.Duration`, thoughts?